Makefile

```diff
@@ -0,0 +1,21 @@
+include .env
+
+# The target directory is used for setting where the output zip files will end up
```

Comment:
Just for info:

Real xmage bundle must be a single zip file with both client and server folders, see `./Utils/build-and-package.pl` as build script example.

Current beta server uses another scripts to build and prepare (folders only for updates and additional launcher files for full bundle):
![shot_231009_092952](https://github.com/magefree/mage/assets/8344157/1825bf61-0b4c-4e44-a8ba-e91837a659da)
![shot_231009_092946](https://github.com/magefree/mage/assets/8344157/9f0715df-494e-4230-8210-ca78a7bf6e3b)

Reply:
Added note that the perl script is source of truth and this is just helper